README.md

```diff
@@ -372,37 +389,37 @@ For updating open swaps, I was successfully able to update 5 positions and recre
 For swapping assets, I was successfully able to chain together 4 swap utxos before the redundant executions caused the transaction to exceed the memory limit.
 
 | Number of Swaps Chained | Tx Fee |
-|--|--|
+|:--:|:--:|
 | 1 | 0.313006 ADA |
 | 2 | 0.511202 ADA |
 | 3 | 0.772815 ADA |
 | 4 | 1.114271 ADA |
 
 ---
 ## Upgradability
-Being that users can close their swaps at any time, whenever there is a potential upgrade, users can choose to close their current swaps and recreate them with the new contracts. It is very similar to how Cardano Stake Pool Operators can currently choose which version of `cardano-node` to use. Just like with upgrading the `cardano-node`, the main difficulty with upgrading beacon tokens is that enough users need to upgrade for there to be enough liquidity for the DEX. 
+Upgrades to `cardano-swaps` can propagate through the ecosystem of users in a similarly democratic fashion as SPOs upgrading their pools to a new version of `cardano-node`. Since users can close their swaps at any time, whenever there is a potential upgrade, users can choose to close their current swaps and recreate them with the new contracts. The main challenge here is the bifurcation of liquidity that occurs during upgrade periods. However, the more users there are and the more overall liquidity there is, the more this issue is minimized.
 
 ---
-## Frontend Agnostic
-By using the beacon tokens, it is trivial for any frontend to integrate with Cardano-Swaps. For example, the new Lace wallet by IOHK can integrate the DEX by simply adding support for querying the beacon tokens. They can also add their own user friendly way to create and use swaps. The only requirement is that all frontends agree to use the same beacon token standard. There is no need for risky extensions or dedicated frontends in order to use this DEX.
+## Frontend Agnosticism
+By using the beacon tokens, it is trivial for any frontend to integrate with Cardano-Swaps. For example, the new Lace wallet by IOHK can integrate the DEX by simply adding support for querying the beacon tokens. They can also add their own user friendly way to create and use swaps. The only requirement is that all frontends/users agree to use the same beacon token standard. There is no need for risky extensions or dedicated frontends in order to use this DEX.
 
 ---
 ## Conclusion
-This DEX protocol has all of the desired properties of a DEX:
+The cardano-swaps protocol has all of the desired properties of a highly scalable DEX:
```

Comment:
Need to mention delegation here too.

Reply:
Also since you dropped the list, you should replace the ending colon with a period.